cfsctl/handle.go

```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"log"
+
+	pb "github.com/c-fs/cfs/proto"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+type handlerFunc func(context.Context, pb.CfsClient) error
+
+func rawHandle(ctx context.Context, fn handlerFunc) {
```

Comment:
i am not a big fan of this approach. i think it simply add unnecessary complexity. but we can revisit it later.